docs/icons.md

```diff
@@ -0,0 +1,52 @@
+---
+label: Iconen
+order: 6
+---
+Iconen moeten betekenis geven aan een grafische gebruikersinterface. Iconen zijn een visuele  presentatie van een object, idee of actie. Als dat object, idee of die actie niet direct duidelijk is voor de gebruiker heeft het icoon ook weinig toegevoegde waarde en kan zelfs leiden tot verwarring en frustratie wat de gebruiker hindert bij het uitvoeren van een taak. Het herkennen en begrijpen van een icoon is afhankelijk van een eerdere ervaring van de gebruiker. Iconen kunnen gebruikers helpen taken te herkennen en onthouden. Doordat een standaard gebruik van iconen ontbreekt zijn labels nodig om de betekenis van een icoon te verduidelijken.
+
+## De voordelen van goed ontworpen iconen
+
+- **Een icoon is een goed 'mikpunt':** ze zijn groot genoeg om makkelijk aan te tikken in een gebruikersinterface waarbij je je vinger nodig hebt om te navigeren, maar ze werken ook goed met een muis cursor;
+- **Ze besparen ruimte:** omdat ze compact zijn kunnen meerdere iconen in een relatief smalle ruimte;
+- **Snelle herkenning:** een bekend of makkelijk te herkennen icoon zal sneller een betekenis verduidelijken, soms sneller dan een tekstlink;
+- **Vindbaarheid:** als men leert welke actie geassocieerd is met een bepaald icoon zal de aanwezigheid van het icoon voor sommige gebruikers de mogelijkheid bieden om sneller en efficienter te werken bij herhalende taken;
+- **Iconen zijn visueel aantrekkelijk:** ze kunnen meerwaarde toevoegen aan een design en het beeld esthetisch aantrekkelijker maken.
+
+## Hoe te gebruiken: formaten en afmetingen
+Iconen verdelen we onder in illustratieve iconen en gebruikersinterface iconen. Iconen worden aangeleverd in SVG formaat zodat het moeiteloos kan schalen op diverse schermbreedtes en altijd haarscherp blijft.
+
+**Illustratieve iconen**
+- Pixelformaat: 80x80
+- Format: SVG
+- Kleurgebruik: wordt aangeleverd in 'Grasgroen' `#39870c` rgb(57, 135, 12)
+- W3C aanbeveling: Scalable Vector Graphics (SVG) 1.1 (Second Edition)
+- Download: losse SVGs
+- Toepassing: op de homepage wordt gebruik gemaakt van illustratieve iconen als aanvulling op de veel gekozen werkzaamheden
+- Beschikbaarheid: te downloaden uit onderstaande tabel
+
+**Gebruikersinterface iconen**
+- Pixelformaat: 24x24
+- Format: SVG
+- Kleurgebruik: wordt aangeleverd in zwart, kleur kan bepaald worden via CSS
+- W3C aanbeveling: Scalable Vector Graphics (SVG) 1.1 (Second Edition)
+- Download: omgezet naar font
```

Comment:
Dit is niet waar, de iconen zitten in een SVG spritesheet

Reply:
check